samples/client/petstore/c/unit-test/test_category.c

```diff
@@ -0,0 +1,68 @@
+#ifndef category_TEST
+#define category_TEST
+
+// the following is to include only the main from the first c file
+#ifndef TEST_MAIN
+#define TEST_MAIN
+#define category_MAIN
+#endif // TEST_MAIN
+
+#include <stdlib.h>
+#include <string.h>
+#include <stdio.h>
+#include <stdbool.h>
+#include "../external/cJSON.h"
+
+
+
+#include "../model/category.h"
+category_t* instantiate_category(int include_optional);
+
+
+
+category_t* instantiate_category(int include_optional) {
+  category_t* category = NULL;
+if (include_optional) {
+  category = category_create(
+56
+//primitive
+, // id
```

Comment:
What about putting the comments in the same line as the example value (e.g. `56, // id (primitive)`) ?

Reply:
This is the plan. I created the PR to receive comments while I developed the approach further.

Reply:
👌